src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright 2014-2017 Spotify AB
- * Copyright 2016-2019 The Last Pickle Ltd
+ * Copyright 2017-2017 Spotify AB
```

Comment:
**Question**: Do you know why that was changed?

Reply:
No, I have no idea on that sorry. It wasn't changed manually... Do you want it changed back?

Reply:
I don't care that much, we should probably stop putting years (and even company names) anyway.